docs/LIMITS.md

```diff
@@ -35,6 +35,8 @@ supported:
 - Only buckets with versioning set to **Enabled** are supported
 - Decompressed individual object size must be less than the Fargate task memory
   limit (`DeletionTaskMemory`) specified when launching the stack
+- S3 Objects transitioned to the `GLACIER` or `DEEP_ARCHIVE` storage classes are
```

Comment:
```suggestion
- S3 Objects using the `GLACIER` or `DEEP_ARCHIVE` storage classes are
```

I'd omit "transitioned" since a transition isn't mandatory for Glacier (anymore).